prime-router/src/test/kotlin/azure/BlobAccessTests.kt

```diff
@@ -25,5 +25,51 @@ class BlobAccessTests {
         } catch (e: MalformedURLException) {
             assertThat(e).isNotNull()
         }
+
+        filename = BlobAccess.BlobInfo.getBlobFilename("")
+        assertThat(filename).isEqualTo("")
+    }
+
+    @Test
+    fun `blob info extension`() {
+        var extension = BlobAccess.BlobInfo
+            .getBlobFileExtension("http://127.0.0.1:10000/devstoreaccount1/reports/ready/pima-az-phd.elr/filename.csv")
```

Comment:
You are testing outside of the unit you have to test.  You do not need to test each specific extension here, just with/without extension.